dev.stewlab.PortaWar.json

```diff
@@ -0,0 +1,32 @@
+{
+    "app-id": "dev.stewlab.PortaWar",
```

Comment:
`stewlab.dev` doesn't exist. This app-id doesn't meet the requirements.

Reply:
resolved